packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -56,8 +78,9 @@ async function createSummarizerCore(
 	const summarizerContainer = await loader.resolve(request);
 	await waitForContainerConnection(summarizerContainer);
 
-	const fluidObject: FluidObject<ISummarizer> | undefined =
-		await summarizerContainer.getEntryPoint();
+	// Old loaders will not have getEntryPoint API on the container. So, use getSummarizerBackCompat which
+	// will use request pattern to get the summarizer in these old loaders.
+	const fluidObject = await getSummarizerBackCompat(summarizerContainer);
```

Comment:
>fluidObject [](http://example.com/codeflow?start=7&length=11)

this can no longer be `undefined`, right?

Reply:
Yep, fixed it.